### DISCUSSION TITLE
Logo idea prototype

### DISCUSSION THREAD

**Discussion by suhtai** [kopykat-kt/kopykat#37] [Ideas]
Need to work on the arrows 😅  and I may choose different cats for each node. But here is an idea 🙂 

<img src="https://user-images.githubusercontent.com/260697/192010415-8ad758f7-458f-47be-ad03-daf953ae59af.png" width="100" />

**Comment by serras** [comment]
What about using cats with different emotions? I think Unicode was given us enough emojis por this 😺➡️😽➡️🙀

**Reply by suhtai** [reply]
>I may choose different cats for each node

Yeah, that's the plan hehe 😅

**Comment by suhtai** [comment]
V2:

<img src="https://user-images.githubusercontent.com/260697/192059909-75a1a8b7-d2df-46db-b480-9ac4e4cdc178.png" width="25" />

<img src="https://user-images.githubusercontent.com/260697/192059909-75a1a8b7-d2df-46db-b480-9ac4e4cdc178.png" width="50" />

<img src="https://user-images.githubusercontent.com/260697/192059909-75a1a8b7-d2df-46db-b480-9ac4e4cdc178.png" width="100" />

<img src="https://user-images.githubusercontent.com/260697/192059909-75a1a8b7-d2df-46db-b480-9ac4e4cdc178.png" width="200" />

**Reply by serras** [reply]
Neat! How does this look at smaller size? I'm afraid the cats may be too small...

**Reply by serras** [reply]
What about slightly increasing the size of the cat faces?

**Comment by suhtai** [comment]
> Neat! How does this look at smaller size? I'm afraid the cats may be too small...

I've updated the comment, yeah the heads may be too small and complex for small sizes. I'll try with more abstract shapes.

**Comment by suhtai** [comment]
V3

<img src="https://user-images.githubusercontent.com/260697/193697740-4f7a5f51-b6e3-4aa4-8c2c-8b621a292883.png" width="25" />

<img src="https://user-images.githubusercontent.com/260697/193697740-4f7a5f51-b6e3-4aa4-8c2c-8b621a292883.png" width="50" />

<img src="https://user-images.githubusercontent.com/260697/193697740-4f7a5f51-b6e3-4aa4-8c2c-8b621a292883.png" width="100" />

<img src="https://user-images.githubusercontent.com/260697/193697740-4f7a5f51-b6e3-4aa4-8c2c-8b621a292883.png" width="200" />

**Reply by serras** [reply]
I like the cats better...

**Reply by suhtai** [reply]
Yeah, I think you are right, they feel less corporate. Plus I can't do design 🙈

**Comment by suhtai** [comment]
V4

<img src="https://user-images.githubusercontent.com/260697/193844450-038b601d-fb78-4ae2-9130-9dd5905253c6.png" width="25" />

<img src="https://user-images.githubusercontent.com/260697/193844450-038b601d-fb78-4ae2-9130-9dd5905253c6.png" width="50" />

<img src="https://user-images.githubusercontent.com/260697/193844450-038b601d-fb78-4ae2-9130-9dd5905253c6.png" width="100" />

<img src="https://user-images.githubusercontent.com/260697/193844450-038b601d-fb78-4ae2-9130-9dd5905253c6.png" width="200" />

With guidelines:

<img src="https://user-images.githubusercontent.com/260697/193845143-f899ba81-8948-4faa-8a02-aa1bb816acd0.png" width="200" />

**Reply by serras** [reply]
I think we should make this one the logo! Could you upload them to the repo, with the black on the sides made transparent?
